README.md

```diff
@@ -1,107 +1,207 @@
 # Overview
 
-[![Join the chat at https://gitter.im/BinaryAnalysisPlatform/bap](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/BinaryAnalysisPlatform/bap?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![Build Status](https://travis-ci.org/BinaryAnalysisPlatform/bap.svg?branch=master)](https://travis-ci.org/BinaryAnalysisPlatform/bap) [![docs](https://img.shields.io/badge/doc-v0.9.9-green.svg)](http://binaryanalysisplatform.github.io/bap/api/v0.9.9/Bap.Std.html) [![docs](https://img.shields.io/badge/doc-master-green.svg)](http://binaryanalysisplatform.github.io/bap/api/master/Bap.Std.html)
-
-BAP is a platform for binary analysis. It is written in OCaml, but can
-be used from other languages.
+[![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/BinaryAnalysisPlatform/bap/blob/master/LICENSE)
+[![Join the chat at https://gitter.im/BinaryAnalysisPlatform/bap](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/BinaryAnalysisPlatform/bap?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![docs](https://img.shields.io/badge/doc-1.0.0-green.svg)](http://binaryanalysisplatform.github.io/bap/api/v1.0.0/argot_index.html)
+[![docs](https://img.shields.io/badge/doc-master-green.svg)](http://binaryanalysisplatform.github.io/bap/api/master/argot_index.html)
+[![Build Status](https://travis-ci.org/BinaryAnalysisPlatform/bap.svg?branch=master)](https://travis-ci.org/BinaryAnalysisPlatform/bap)
+[![pip](https://img.shields.io/badge/pip-1.1.0-green.svg)](https://pypi.python.org/pypi/bap/)
+
+Binary Analysis Platform is a framework for writing program analysis
+tools, that target binary files. The framework consists of a plethora
+of libraries, plugins, and frontends. The libraries provide code
+reusability, the plugins facilitate extensibility, and the frontends
+serve as entry points.
 
 # <a name="Installation"></a>Installation
 
-BAP is released using `opam` package manager. After you've successfully
-[installed](https://opam.ocaml.org/doc/Install.html) opam, do the following:
+We use the OPAM package manager to handle installation. After you've
+successfully [installed](https://opam.ocaml.org/doc/Install.html)
+opam, do the following:
```

Comment:
There is two ways to spell "opam".

[OPAM](), which appears to be what the OPAM folks stylize their project.

`opam`, which refers to the command / binary.

I think here we can mean either. But for lines 72 and 248, I think we really mean OPAM.